doc/c128.sgml

```diff
@@ -200,15 +200,19 @@ memory drivers using the VDC memory!
 <descrip>
   <tag><tt/c128-vdc.tgi (c128_vdc_tgi)/</tag>
```

Comment:
The predominant style seems to be to have blank lines after `<descrip>` and before `</descrip>`. I see that you follow that style for newly added descriptions. It would be nice if you would additionally modify descriptions you edit anyway to follow that style too.